tests/scripts/start-cypress-tests

```diff
@@ -40,14 +40,15 @@ docker run -v $PWD:/workdir -w /workdir                     \
 # Move elemental.iso into the expected folder
 if [[ ! -f ${ELEMENTAL_ISO} ]]; then
   # Upgrade scenario
-  # We start from stable ISO built within RegistrationEndpoint menu
-  # Because we have to test the download feature but we can only build from stable for now
-  if [[ ${CYPRESS_TAGS} == "upgrade" ]]; then
-    mv downloads/*.iso ${ELEMENTAL_ISO}
+  # We need to build the ISO with stable base image
+  # It can not be done from the UI for now
+  
   # RKE2 as upstream cluster
   # We cannot use PXE boot so we have to boot from ISO but we cannot use stable ISO
   # We need to use a custom ISO and the menu doesn't allow to do it now.
-  elif grep rke <<< ${K8S_UPSTREAM_VERSION}; then
+  if [[ ${CYPRESS_TAGS} == "upgrade" ]] || grep rke <<< ${K8S_UPSTREAM_VERSION}; then
+    # Wait a bit to let the ISO be built (could be improved)
+    sleep 180
```

Comment:
How us the ISO built? Maybe checking the filesize change can be an option? But anyway for another PR.